pkg/log/json.go

```diff
@@ -193,7 +193,9 @@ func (w *JSONWriter) Fail(format string, args ...interface{}) {
 	log.out.Infof(format, args...)
 	msg := fmt.Sprintf("%s %s", errorSymbol, fmt.Sprintf(format, args...))
 	if msg != "" {
-
+		if log.stage == "" {
+			log.stage = "Internal server error"
```

Comment:
I don't like too much to show `internal server error` as stage, maybe `unexpected error`. This is what we'll display in the UI when an error occurs outside a defined stage.

cc @pchico83 @rlamana

Reply:
the absence of stage is always meaning is an error? in that case, the stage could be called `errors` ? 🤔 💭

Reply:
Well, it is an error because the function is `Fail` and should be called only when there is an error.

The name we use for the stage, it will be the one that appears in the UI when an unexpected error happens, not sure `errors` would be a meaningful text to show